planetarium/NineChronicles
# Update l10n font assets

### Description
1. Fix `ChineseSimplified` font problems.

### How to test
1. Change `Game` scene settings like below. 
![image](https://user-images.githubusercontent.com/6128868/123749335-479aa580-d8f0-11eb-82ee-f51666d20294.png)
1. Play `Game` scene and check out relative tasks.

### Related Links
1. [[UI] 중국어(간체)로 실행 시 전투 준비 화면 내 AP 소모량이 노출되지 않는 현상](https://app.asana.com/0/1141562434100787/1200457522197535/f)
1. [언어별 폰트 누락되어 네모로 나오는 현상](https://app.asana.com/0/1133453747809944/1200345696716771/f)

### Screenshot
![image](https://user-images.githubusercontent.com/6128868/123749052-f68ab180-d8ef-11eb-977e-5ab80f94e35e.png)
![image](https://user-images.githubusercontent.com/6128868/123749096-04d8cd80-d8f0-11eb-9fd1-271c326911a4.png)



┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1141562434100787/1200534818733570) by [Unito](https://www.unito.io)
